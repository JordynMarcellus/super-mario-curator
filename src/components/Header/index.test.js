import React from 'react'
import {render} from '@testing-library/react'
import {Header} from './'

describe('Header component tests', () => {
    it('renders a component with arbitrary children', () => {
        const {container} = render(
            <Header>
                <h1>Hi! I'm the header's child.</h1>
            </Header>
        )
        expect(container.firstChild).toMatchSnapshot()
        expect(container).toHaveTextContent("Hi! I'm the header's child.")
    })
})