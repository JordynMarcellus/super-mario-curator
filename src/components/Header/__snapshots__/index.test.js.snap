// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component tests renders a component with arbitrary children 1`] = `
<header
  class="StyledBox-sc-13pk1d4-0 crKXlR"
  style="z-index: 1;"
>
  <h1>
    Hi! I'm the header's child.
  </h1>
</header>
`;
